src/utils/datetime.js

```diff
@@ -110,3 +110,18 @@ export function millisecondsAsTimeOfDay(milliseconds, format = 'h:mm a') {
   }
   return moment.utc(milliseconds).format(format);
 }
+
+/**
+ * formatDisplayDate
+ * @param  {Number} timestamp integer hammertime
+ * @param  {Object} timePrefs object containing timezoneAware Boolean and timezoneName String
+ *
+ * @return {String}           formatted timezoneAware date string
+ */
+export function formatDisplayDate(timestamp, timePrefs) {
+  if (!timestamp) {
+    return '';
+  }
+  return moment(timestamp).utc().tz(getTimezoneFromTimePrefs(timePrefs))
```

Comment:
hmrt is defined in the second line - abbr. of hammertime, result of Date.parse(time)